internal/netpoll/kqueue_events.go

```diff
@@ -28,7 +28,7 @@ const (
 	// InitEvents represents the initial length of poller event-list.
 	InitEvents = 64
 	// AsyncTasks is the maximum number of asynchronous tasks that the event-loop will process at one time.
-	AsyncTasks = 48
+	AsyncTasks = 256
```

Comment:
May want to comment why kqueue and epoll has different values for it?
Also I'd suggest to use a more descriptive name like `AsyncTasksInOneLoop` etc.